migrations/versions/4bca517e56a0_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: 4bca517e56a0
+Revises: 
+Create Date: 2021-10-22 13:06:12.466269
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '4bca517e56a0'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('planet',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('name', sa.String(), nullable=True),
+    sa.Column('description', sa.String(), nullable=True),
+    sa.Column('moons', sa.Integer(), nullable=True),
```

Comment:
Notice that since your model definitions didn't say that these fields were required, the migration marks them as being OK to be null.